<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1406</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="table_barname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1111</width>
      <height>541</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="lineWidth">
     <number>20</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="el_name">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>640</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>نام درس</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="el_tname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>نام استاد درس</string>
    </property>
   </widget>
   <widget class="QComboBox" name="day1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>720</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>شنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>یکشنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>دوشنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>سه شنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>چهار شنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>پنجشنبه</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="hafte1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>هر هفته</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>زوج</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>فرد</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="hafte2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>هر هفته</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>زوج</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>فرد</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="button_addlesson">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>331</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>ثبت</string>
    </property>
   </widget>
   <widget class="QComboBox" name="time1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>720</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>۸</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۰</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۲</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۸</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="day2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>760</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>شنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>یکشنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>دوشنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>سه شنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>چهار شنبه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>پنجشنبه</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="time2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>760</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>۸</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۰</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۲</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۸</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="el_code">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>کد درس</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="el_search">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>600</y>
      <width>701</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>جست و جو</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>931</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>ذخیره برنامه</string>
    </property>
   </widget>
   <widget class="QComboBox" name="group">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>680</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>گروه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۲</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۳</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۵</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۷</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۸</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۹</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۰</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۱</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۲</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۳</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۵</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۷</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۸</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱۹</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="vahed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>واحد</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۱</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۲</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۳</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>۵</string>
     </property>
    </item>
   </widget>
   <widget class="QTableView" name="table_lesson">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>640</y>
      <width>1021</width>
      <height>151</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_removelesson">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>800</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>حذف</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add2barname">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>800</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>اضافه کن</string>
    </property>
   </widget>
   <widget class="QTableView" name="table_entekhabshode">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>251</width>
      <height>541</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="buttom_remove_entekhabshode">
    <property name="geometry">
     <rect>
      <x>1137</x>
      <y>560</y>
      <width>241</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>حذف</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_show">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>610</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>عدم نمایش دروسی که امکان انتخاب وجود ندارد</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1037</x>
      <y>560</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>جمع واحد ها :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_jame">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>560</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1406</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
